insights/specs/default.py

```diff
@@ -484,6 +484,8 @@ class DefaultSpecs(Specs):
     ose_node_config = simple_file("/etc/origin/node/node-config.yaml")
     ovirt_engine_server_log = simple_file("/var/log/ovirt-engine/server.log")
     ovirt_engine_ui_log = simple_file("/var/log/ovirt-engine/ui.log")
+    ovs_vsctl_list_br = simple_command("/usr/bin/ovs-vsctl list-br")
```

Comment:
@psachin these also need to be added to the JSON files for collection.  The `ovs_vsctl_list_br` command needs to be added to insights/specs/insights_archive.py.

Reply:
Nevermind, now I see what you are doing.